dev-php/pecl-memcache/pecl-memcache-8.0.ebuild

```diff
@@ -23,9 +21,7 @@ SLOT="8"
 IUSE="+session"
 
 DEPEND="php_targets_php8-0? ( sys-libs/zlib )"
-RDEPEND="${DEPEND}
-	php_targets_php7-4? ( dev-php/pecl-memcache:7[php_targets_php7-4(-)?] )
-"
+RDEPEND="${DEPEND}"
 
 # The test suite requires memcached to be running.
 RESTRICT='test'
```

Comment:
I really don't like these weird if ... else default constructs

I wonder if it was a hack to transition to newer behaviour, but they did it based on PHP version instead of some eclass variable or EAPI?

Reply:
`pecl-memcache-8.0-r1.ebuild` is fine, `pecl-memcache-4.0.5.2-r1.ebuild` is completely broken

Reply:
removed `pecl-memcache-4.0.5.2-r1.ebuild`